src/translib/transformer/xfmr_intf.go

```diff
@@ -1703,6 +1731,20 @@ var populatePortCounters PopulateIntfCounters = func (inParams XfmrParams, count
                 err = errors.New("Get Counter URI failed")
             }
         }
+    case "/openconfig-interfaces:interfaces/interface/ethernet/state/counters",
+         "/openconfig-interfaces:interfaces/interface/openconfig-if-ethernet:ethernet/state/counters":
+        for _, attr := range etherCntList {
+            uri := targetUriPath + "/" + attr
+            if ok, err := getSpecificCounterAttr(uri, &CounterData, &CounterBackUpData, counter); !ok || err != nil {
+                log.Info("Get Ethernet Counter URI failed :", uri)
+                err = errors.New("Get Ethernet Counter URI failed")
```

Comment:
Do we need to return here or continue processing?

Reply:
This loop is for all counters, if one fails will report the log and continue filling other attributes.